.github/CODEOWNERS

```diff
@@ -0,0 +1,7 @@
+# Users referenced in this file will automatically be requested as reviewers for PRs that modify the given paths.
+# See https://help.github.com/articles/about-code-owners/
+
+/eng/                                   @aspnet/build
```

Comment:
We should change these code ownerships. It should be us.

Reply:
This file was not intended to stay. We're not making any changes in eng/ other than in Versions.props and Versions.Details.xml. These changes are minor and we don't really need to be requesting for reviews there.